src/citrine/resources/design_space.py

```diff
@@ -3,28 +3,16 @@
 from typing import Iterable, Optional, TypeVar, Union
 from uuid import UUID
 
-from gemd.enumeration.base_enumeration import BaseEnumeration
 
 from citrine._utils.functions import format_escaped_url
-from citrine.informatics.design_spaces import DesignSpace, EnumeratedDesignSpace, \
-    HierarchicalDesignSpace
+from citrine.informatics.design_spaces import DefaultDesignSpaceMode, DesignSpace, \
+    DesignSpaceSettings, EnumeratedDesignSpace, HierarchicalDesignSpace
 from citrine._rest.collection import Collection
 from citrine._session import Session
 
 CreationType = TypeVar('CreationType', bound=DesignSpace)
 
 
-class DefaultDesignSpaceMode(BaseEnumeration):
```

Comment:
Although the class definition is moved, since it's still being imported here, it will not break user imports.